### ISSUE TITLE
Scalafix from 0.18 to 0.19

### ISSUE THREAD

**Issue by rossabaker** [http4s/http4s#2045]
* 0.18 to 0.19 has a lot of mechanical changes that would be awfully nice to implement via scalafix.
* 0.19 to 1.0 will likely have more, and it'd be nice to have a precedent.

Whoever does this will be Great Friend of Project. :medal_military:

**Comment by Zelenya** [comment]
Idea.
1. List all the changes which need a scalafix (in this issue or separate issues)
2. Tweet a call for participation
3. Wake up to everything fixed

**Comment by jmcardon** [comment]
> Wake up to everything fixed

_Someday, I wish upon a star_
_Wake up where the clouds are far behind me_

**Comment by rossabaker** [comment]
I think that's a great idea.  A couple that immediately to mind:
* `withBody` => `withEntity` (tricky, because return type changes)
* `HttpService => HttpRoutes.of`
* The client builders (after #1993)

I'm sure I'll think of more as I update release notes.

What else are upgraders suffering from?

**Comment by hamnis** [comment]
add typealiases `RequestCookie` and `ResponseCookie` in `org.http4s` to `org.http4s.Cookie` in 0.18.x may help a bit.

**Comment by m-sp** [comment]
I think this got pretty stale, we have Scalafix 0.18 to 0.20
